check.go

```diff
@@ -170,14 +174,14 @@ func (c *CheckRunner) updateCheckTCP(latestCheck *api.HealthCheck, checkHash typ
 			return false
 		}
 
-		c.logger.Printf("[INFO] Updating TCP check %q", checkHash)
+		c.logger.Info("Updating TCP check ", "checkHash", hclog.Fmt("%q", checkHash))
 
 		if tcpCheckExists {
 			tcpCheck.Stop()
 		} else {
 			httpCheck, httpCheckExists := c.checksHTTP[checkHash]
 			if !httpCheckExists {
-				c.logger.Printf("[WARN] Inconsistency check %q - is not TCP and HTTP", checkHash)
+				c.logger.Warn("Inconsistency check ", "checkHash", hclog.Fmt("%q - is not TCP and HTTP", checkHash))
```

Comment:
Similar to my previous suggestion, this could be reworded to make better use of the k/v pairs:
```
c.logger.Warn("Inconsistency check is not TCP and HTTP", "checkHash", hclog.Fmt("%q", checkHash))
```